### PR TITLE
{lib, data, numlib, tools, math}[GCCcore/13.3.0, foss/2024a, gfbf/2024a] libmad-0.15.1b, SoX-14.4.2,  SuiteSparse-7.8.2, Szip-2.1.1, sympy-1.13.2, trimesh-4.4.9, Voro++-0.4.6, Xerces-C++-3.2.5, zsh-5.9

### PR DESCRIPTION
**Pull request by ankekreuzer**
Another bunch of packages from the new SW stage at JSC. And again, mostly versions bumps.

### CODE REVIEW

**Comment by ankekreuzer** [comment]
Test report by @ankekreuzer
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in total)
zam329 - Linux Rocky Linux 9.4 (Blue Onyx), x86_64, Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz, Python 3.9.18
See https://gist.github.com/ankekreuzer/73c323d7c35f007dae04ec4077d7ddb8 for a full test report.

File: easybuild/easyconfigs/z/zsh/zsh-5.9-GCCcore-13.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+checksums = ['9b8d1ecedd5b5e81fbf1918e876752a7dd948e05c1a0dba10ab863842d45acd5']
+
+builddependencies = [
+    ('binutils', '2.42'),
+]
+
+dependencies = [
+    ('ncurses', '6.5'),
+]
+
```
Comment (smoors) [inline easybuild/easyconfigs/z/zsh/zsh-5.9-GCCcore-13.3.0.eb:25]: sanity_check_paths is missing here.
can you also add sanity_check_commands?
it' probably the same as in zsh-5.9.eb:
```
sanity_check_paths = {
    'files': ['bin/zsh'],
    'dirs': ['lib/zsh/%(version)s', 'share'],
}

sanity_check_commands = ['zsh --version']
```

Comment (ankekreuzer) [reply easybuild/easyconfigs/z/zsh/zsh-5.9-GCCcore-13.3.0.eb:25]: Oh, yes, sorry.
It is now added.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
this is the zsh build error:
```
checking if tcsetpgrp() actually works... notty
configure: error: no controlling tty
Try running configure with --with-tcsetpgrp or --without-tcsetpgrp
```

try adding `configopts = '--without-tcsetpgrp'`?

**Comment by ankekreuzer** [comment]
Sorry that this took so long. I added the configopts now.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen3

**Comment by ankekreuzer** [comment]
Ah, then I will just remove sympy from this MR since it is already merged with a newer version.

**Comment by SebastianAchilles** [comment]
Szip 2.1.1 was merged in https://github.com/easybuilders/easybuild-easyconfigs/pull/21510 The content is the same, so I would suggest we remove it from this PR.

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
bwd-rockylinux-94 - Linux Rocky Linux 9.4 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), 2 x NVIDIA NVIDIA GeForce GTX 1060 6GB, 535.183.06, Python 3.9.18
See https://gist.github.com/SebastianAchilles/ff47795d7ebcb6902e46c4921071916d for a full test report.

**Comment by SebastianAchilles** [comment]
Going in, thanks @ankekreuzer!

**Pull request merged**